beacon-chain/core/epoch/epoch_operations_test.go

```diff
@@ -478,13 +478,19 @@ func TestInclusionSlotOk(t *testing.T) {
 		{Data: &pb.AttestationData{},
 			ParticipationBitfield: []byte{0xFF},
 			SlotIncluded:          100},
+		{Data: &pb.AttestationData{},
+			ParticipationBitfield: []byte{0xFF},
+			SlotIncluded:          99},
+		{Data: &pb.AttestationData{},
+			ParticipationBitfield: []byte{0xFF},
+			SlotIncluded:          88},
 	}
 	slot, err := InclusionSlot(state, 45)
 	if err != nil {
 		t.Fatalf("Could not execute InclusionSlot: %v", err)
 	}
 	// validator 45's attestation got included in slot 100.
-	if slot != 100 {
+	if slot != 88 {
```

Comment:
Why change? This doesn't reflect the comment above

Reply:
updated to 100. slot={101,100,102) hence it chooses 100e